rpc_util.go

```diff
@@ -808,6 +809,9 @@ func (p *payloadInfo) free() {
 	}
 }
 
+// Global error for message size exceeding the limit
+var ErrMaxMessageSizeExceeded = errors.New("max message size exceeded")
```

Comment:
Don't export this type, make the first letter lowercase, otherwise it will become part of the gRPC's public API. Comment should start with the name of the variable: `errMaxMessageSizeExceeded  represents an error due to exceeding the maximum message size limit`.